development/prow-installer/cmd/installer/main.go

```diff
@@ -60,7 +65,7 @@ func main() {
 		ServiceAccount: *credentialsFile,
 	}
 
-	//TODO: we need to refactor cluster package or installer package to not pass credentialsfiles data twice in a call to NewCLient method.
+	//TODO: refactor GKE operating packages to use os.Getenv("GOOGLE_APPLICATION_CREDENTIALS") in order to get credentials.
```

Comment:
We can remove this TODO, right? Above you already set the env variable to whatever is in `*credentialsFile`.

Reply:
I changed this TODO beacause the packages use passed `*credentialsFile` variable to make clients authenticate to GCloud. We can change the authentication to using environment variable instead of passing the value.